### PR TITLE
Fixed non aligned tags in account forms

### LINKED ISSUE #561: Edit Profile Page Tags not aligned

### PR DESCRIPTION
**Pull request by eevanwong**
…r and admin account forms

## Ticket link

Closes #561 

## Steps to test
1. Go to volunteer account, check edit account profile
2. Go to admin account , check edit account profile
3. Check admin and volunteer create account forms

Volunteer Account Form
image

Employee Account Form
image

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/user/AccountForm.tsx [hunk 1]
```diff
@@ -389,21 +389,26 @@ const AccountForm = ({
                 isRequired
               />
```
Comment (MatoPlus) [inline frontend/src/components/user/AccountForm.tsx:390]:
```suggestion
              />
<Box />
```

File: frontend/src/components/user/AccountForm.tsx [hunk 2]
```diff
@@ -389,21 +389,26 @@ const AccountForm = ({
...
+                    onSelect={(skill) =>
+                      selectSkill(skill, values.skills, setFieldValue)
+                    }
+                    onDeselect={(skill) =>
+                      deselectSkill(skill, values.skills, setFieldValue)
+                    }
+                  />
+                </>
+              ) : (
+                <Box />
```
Comment (MatoPlus) [inline frontend/src/components/user/AccountForm.tsx:411]: Can we just always render an empty box before to avoid this ternary?

**Review by MatoPlus — CHANGES_REQUESTED** [review]
LGTM with suggestion

File: frontend/src/components/user/AccountForm.tsx (hunk 1 above)
Comment (eevanwong) [reply frontend/src/components/user/AccountForm.tsx:390]: Yep you're correct. This is better than what I originally had.

**Review by MatoPlus — APPROVED** [review]
LGTM

**Pull request merged**